cabal.project

```diff
@@ -200,3 +213,7 @@ constraints:
 
 package comonad
   flags: -test-doctests
+
+allow-newer:
+  monoidal-containers:aeson,
```

Comment:
You may not need `monoidal-containers:aeson` anymore as of now.

https://github.com/input-output-hk/cardano-ledger-specs/pull/2245